src/constants/page.js

```diff
@@ -0,0 +1,11 @@
+// TODO : HOME(/)과 PRODUCT_LIST(/product-list) 구분해야할지
+// TODO : createBrowserRouter vs BrowserRouter - createBrowserRouter로 useNavigate 적용할 수는 없나?
+
+export const ROUTE_PATH = {
+  PRODUCT_LIST: "/",
+  PRODUCT_DETAIL: "/product-detail",
+  CART: "/carts",
+  ORDER: "/orders",
+  PAYMENT: "/payment",
+  ORDER_DETAIL: "/order-detail",
```

Comment:
```suggestion
  PRODUCT_DETAIL: "/product-detail", // `/product/${product_id}`
  CART: "/carts",
  ORDER: "/orders",
  PAYMENT: "/payment",
  ORDER_DETAIL: "/order-detail",  // `/order/${order_id}`
```

일반적으로는 위처럼 뒤에 path param으로 상품이나 주문 id가 들어갈 것 같아요

Reply:
디테일 체크 감사합니다! - [079d528](https://github.com/next-step/react-shopping-cart/pull/29/commits/079d52804a7936022cf87fc2defd5909f0e34f3f)